js-weather/weather.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>現在のお天気</title>
+    <link rel="stylesheet" href="stylesheet.css">
+    <link href="https://fonts.googleapis.com/earlyaccess/kokoro.css" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="container">
+            <img src="img/k0500_0.png" alt="傘を持った男性">
+            <img src="img/k0500_1.png" alt="傘を持った男性">
+            <img src="img/k0500_2.png" alt="傘を持った女性">
+            <p>現在の天気</p>
+            <img src="img/k0500_3.png" alt="傘を持った男性">
+            <img src="img/k0500_4.png" alt="傘を持った女性">
+            <img src="img/k0500_5.png" alt="傘を持った男性">
+        </div>
+    </header>
+    <main>
+        <section class="container">
+            <h1>世界各国の天気を見よう</h1>
+            <form>
+                <label for="cityId">地域を選んでください。</label>
+                <select id="cityId">
+                    <option value="2643743" selected>ロンドン</option>
+                    <option value="1850147" >東京</option>
```

Comment:
（修正不要）
余計な空白が入っています

```suggestion
                    <option value="1850147">東京</option>
```